src/test/java/internship/tests/PersonalDetailsTest.java

```diff
@@ -0,0 +1,60 @@
+package internship.tests;
+
+import internship.framework.core.TestBase;
+import internship.framework.core.runner.GlobalProperties;
+import internship.framework.pages.LoginPage;
+import internship.framework.pages.PersonalDetailsPage;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class PersonalDetailsTest extends TestBase {
+    private ThreadLocal<SoftAssert> softAssert = new ThreadLocal<>();
+
+    @Test(testName = "Personal details test",
+            description = "Testing personal details.")
+    public void checkPersonalDetailsFunctionality() {
+        softAssert.set(new SoftAssert());
```

Comment:
Can we move line 17,18 from the test method somewhere else? This doesn't look related for the test.

Reply:
Line 17 . Cristina's advice was to set a new SoftAssert object at the begining of every test and remove it after every testing, using tearDown method.
Line 18. login() method is part of this test requirement.

Reply:
**Line 17 . Cristina's advice was to set a new SoftAssert object at the begining of every test and remove it after every testing, using tearDown method.** So, how can we do this for line 17? Instead of copy/pasting the same line for each test?

Reply:
Line 18 can remain. If its part only for this test.